src/common/lib/categories.js

```diff
@@ -0,0 +1,6 @@
+import axios from "axios"
+const baseUrl = "https://backend-dev.debio.network"
```

Comment:
I think you need to register this on the env file and call here. never do this way

Reply:
done mas diki, kindly re-review :)